backend-project/small_eod/users/models.py

```diff
@@ -2,4 +2,8 @@
 
 
 class User(AbstractUser):
-    pass
+    def notify(self, sender, action, **kwargs):
```

Comment:
```
enabled = self.get_enabled_notifications();
key = f`${sender}_${self.action}`
if key not in enabled:
   return false
return TemplateMailManager.send(key, ....)
```

Zauważ, że dzieki takiemu podejściu wyraźnie widać, że to użytkownik decyduje jakie powiadomienia są dla niego wysyłane.